.eslintrc

```diff
@@ -9,6 +9,17 @@
         "before": "readonly",
         "beforeEach": "readonly"
       }
+    },
+    {
+      "files": ["**/*.js"],
+      "excludedFiles": ["bin/*"],
```

Comment:
I think you should be able to do:
```suggestion
      "excludedFiles": ["bin"],
```

Reply:
I've tried doing this initially but it doesn't work